content/chainguard/chainguard-images/getting-started/nginx/index.md

```diff
@@ -0,0 +1,262 @@
+nano stylesheet.css
+```
+
+Copy the following code to your `stylesheet.css` file. 
+
+```CSS
+/*
+Chainguard Academy nginx Demo Application
+*/
+
+body {
+    text-align: center;
+    background-color: #fcebfc;
+    font-family: Arial, sans-serif;
+}
+
+h1 {
+    color: #df45e6;
+}
+
+h2, h3 {
+    color: #9745e6;
+}
+
+p {
+    color: #583ce8
+}
+
+.corners {
+    border-radius: 10px;
+}
+
+```
+
+After copying the code into the `stylesheet.css` file, save and close it.
+
+Next, you will pull down the `inky.png` file using `curl`. Always [inspect the URL](https://raw.githubusercontent.com/chainguard-dev/edu-images-demos/734e6171ee69f0e0dbac95e6ebc1759ac18bf00a/nginx/data/inky.png) before downloading it to ensure it comes from a safe and trustworthy location.
+
+```shell
+curl -O https://raw.githubusercontent.com/chainguard-dev/edu-images-demos/734e6171ee69f0e0dbac95e6ebc1759ac18bf00a/nginx/data/inky.png
+```
+
+Now, return to the `nginxdemo` directory you made earlier. Here we will create the `nginx.conf` configuration file used by nginx to run the local web server. We will demonstrate this using `nano`.
+
+```shell
+nano nginx.conf
+```
+
+Copy the following code into the configuration file you created. In the `location` directive, be sure to update the configuration to reference the path from root to the `data` directory you created in a previous step.
+
+```nginx
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    server {
+
+        listen 1313;
+        server_name localhost;
+        charset koi8-r;
+
+        location / {
+            root /Users/username/nginxdemo/data; # Update the file path for your system
+
+        }
+
+    }
+
+    include servers/*;
+
+}
+```
+
+Once you are finished editing the configuration file, save and close it.
+
+Create a new file named `mime.types` using a text editor of your choice. We will use `nano` to demonstrate.
+
+```shell
+nano mime.types
+```
+
+Copy and paste the following code into your `mime.types` file. This file will allow nginx to handle the HTML, CSS, and png files we created when rendering the webserver. 
+
+```nginx
+types {
+    text/html                                        html htm shtml;
+    text/css                                         css;
+    image/png                                        png;
+}
+```
+
+Save and close the `mime.types` file when you are finished editing it.
+
+Next, copy the filepath to the `nginx.conf` file you created earlier. Replacing the example path with this copied path, execute the following command to initialize the nginx server:
+
+```shell
+nginx -c /Users/username/nginxdemo/nginx.conf
+```
+
+To view the HTML content, navigate to `localhost:8080` in your web browser of choice. You should see a simple landing page with a picture of Inky and a fun fact about the Wolfi octopus.
```

Comment:
Didn't you specify listen 1313 as a directive in the config? I think it shouldn't be appearing at 80. I'd triple check this (make sure your default config isn't loading, kill all nginx processes, run it again and see what port it appears on), as it should be appearing on 1313 and not 80 (I think).

Reply:
Good catch, thanks! I wanted it to be 8080, I think I changed to 1313 for testing at some point